src/cacheHttpClient.ts

```diff
@@ -253,33 +264,35 @@ async function uploadFile(
 }
 
 async function commitCache(
-    restClient: RestClient,
+    httpClient: HttpClient,
     cacheId: number,
     filesize: number
-): Promise<IRestResponse<void>> {
-    const requestOptions = getRequestOptions();
+): Promise<ITypedResponse<null>> {
     const commitCacheRequest: CommitCacheRequest = { size: filesize };
-    return await restClient.create(
-        `caches/${cacheId.toString()}`,
+    const additionalHeaders = {
+        "Content-Type": "application/json"
+    };
+    return await httpClient.postJson<null>(
```

Comment:
Should this be `void` instead of `null`?

Reply:
I wasnt totally sure. the `ITypedResponse.result` is type `T | null` so using `void` still had the result type as `void | null` which didnt seem more useful than just null